settings.php

```diff
@@ -15,162 +15,48 @@
-    $desc = get_string('aimedatstudents_desc', 'tiny_c4l');
-    $setting = new admin_setting_configmulticheckbox('tiny_c4l/aimedatstudents', $name, $desc, $components, $components);
-    $settings->add($setting);
-
-
-    // Configure not intended students' components.
-    $components = [
-        'estimatedtime' => get_string('estimatedtime', 'tiny_c4l'),
-        'duedate' => get_string('duedate', 'tiny_c4l'),
-        'proceduralcontext' => get_string('proceduralcontext', 'tiny_c4l'),
-        'gradingvalue' => get_string('gradingvalue', 'tiny_c4l'),
-        'expectedfeedback' => get_string('expectedfeedback', 'tiny_c4l'),
-        'learningoutcomes' => get_string('learningoutcomes', 'tiny_c4l'),
-    ];
-    $name = get_string('notintendedforstudents', 'tiny_c4l');
-    $desc = get_string('notintendedforstudents_desc', 'tiny_c4l');
-    $setting = new admin_setting_configmulticheckbox('tiny_c4l/notintendedforstudents', $name, $desc, [], $components);
-    $settings->add($setting);
-
-    // Custom components settings.
-    $settings->add(new admin_setting_heading('tiny_c4l/customcomponents', new lang_string('customcomponents', 'tiny_c4l'), ''));
-
-    // Number of custom components.
-    $name = 'tiny_c4l/customcompcount';
-    $title = get_string('customcompcount', 'tiny_c4l');
-    $description = get_string('customcompcountdesc', 'tiny_c4l');
-    $options = range(0, 12);
-    $options = array_combine($options, $options);
-    $setting = new admin_setting_configselect($name, $title, $description, 0, $options);
-    $setting->set_updatedcallback('purge_all_caches');
-    $settings->add($setting);
-
-
-    // CSS for preview content inside editor.
-    $name = 'tiny_c4l/custompreviewcss';
-    $title = get_string('custompreviewcss', 'tiny_c4l');
-    $url = new moodle_url('/admin/settings.php', ['section' => 'additionalhtml']);
-    $link = html_writer::link($url, get_string('additionalhtml', 'tiny_c4l'), ['target' => '_blank']);
-    $description = get_string('custompreviewcssdesc', 'tiny_c4l', $link);
-    $setting = new admin_setting_configtextarea($name, $title, $description, '');
+    $setting = new admin_setting_configcheckbox('tiny_elements/enablepreview', $name, $desc, $default);
     $settings->add($setting);
 
-    // Images bank.
-    $fileareaid = 'customimagesbank';
-    $name = 'tiny_c4l/customimagesbank';
-    $title = get_string('customimagesbank', 'tiny_c4l');
-    $description = get_string('customimagesbankdesc', 'tiny_c4l');
-    $options = ['accepted_types' => ['image'], 'maxfiles' => -1];
-    $setting = new admin_setting_configstoredfile($name, $title, $description, $fileareaid, 0, $options);
+    // Mark inserted components.
+    $name = get_string('markinserted', 'tiny_elements');
+    $desc = get_string('markinserted_desc', 'tiny_elements');
+    $default = 1;
+    $setting = new admin_setting_configcheckbox('tiny_elements/markinserted', $name, $desc, $default);
```

Comment:
does not work as intended (wraps wrongly when inserting), pls fix or remove feature completely

Reply:
Removed